cmake/modules/contrib/EthosU.cmake

```diff
@@ -18,7 +18,8 @@
 if(USE_ETHOSU)
   file(GLOB COMPILER_ETHOSU_SRCS
        CONFIGURE_DEPENDS src/relay/backend/contrib/ethosu/*
-       CONFIGURE_DEPENDS src/contrib/ethosu/cascader/*)
+       CONFIGURE_DEPENDS src/contrib/ethosu/cascader/*
```

Comment:
If we have source files that defines object definitions, we need to include the irrespective of cmake config.
See here : https://github.com/apache/tvm/pull/9630

Reply:
Understood, good catch. I've added these files to the 'else' condition here.